### ISSUE TITLE
AE2 glitch

### ISSUE THREAD

**Issue by Or-Gaule** [xJon/The-1.7.10-Pack#1988]
Hello.
So whenever I visit other dimensions, be it the nether, the last millenium or even the mining dimension and then come back to the overworld, all of my autocrafting patterns in the ME system get all glitched out, some patterns get deleted altogether. 
![2022-12-06_17 01 41](https://user-images.githubusercontent.com/119957996/205963903-c4a26457-4379-467c-83e4-c72a9542c55b.png)
![2022-12-06_17 01 31](https://user-images.githubusercontent.com/119957996/205964191-ad668d44-7e4a-4b91-9eac-bf2fa7a8df67.png)
![2022-12-06_17 01 07](https://user-images.githubusercontent.com/119957996/205964230-f62a3d5c-1909-45f2-9e02-1df43077ce48.png)
So basically the patterns seem to replicate, to the point there are several rows of the same patterns, one row pretty much gets duplicated. Needless to say autocrafting doesn't work afterwards. Sometimes the corresponding machines get corrupted, for example a fully upgraded induction smelter loses all of its upgrades, reverts back to the basic version and the side configuration is all over the place.

It goes back to normal once I relog, but it is pretty impractical to have to relog everytime... 

I'm playing on a singleplayer world.

Thanks for all the ideas.

**Comment by xJon** [comment]
Some players reported glitches when doing dimensional travel that were resolved with a modpack reinstall. Could you try that?

**Comment by Or-Gaule** [comment]
Unfortunately that didn't work

**Comment by xJon** [comment]
Did you do a full clean reinstall or only a normal one through the launcher? If you didn't, I'd recommend you to try that first and copy over your save, as that usually resolved this issue to players updating from older versions of the modpack. Let me know how it goes!

**Comment by Or-Gaule** [comment]
I did just that. Made a backup of my world and then deleted the 1.7.10 pack folder, is that what you have in mind?

**Comment by xJon** [comment]
Yes, if that didn't work that's unfortunate. I am not sure what is the cause and why that worked for others.
Beyond that, maybe you can share a game log using https://mclo.gs from a session in which this occurs? Maybe there will be another hint for the cause of this.

**Comment by Or-Gaule** [comment]
I thought I'd wait for the modpack update and hopefully it would fix itself

**Comment by xJon** [comment]
Because I'm not sure what mod or mod conflict is causing this, a future update cannot guarantee a solution unfortunately. Sorry about that.

**Comment by Or-Gaule** [comment]
No worries man, it's not that important. I might just try messing with it myself for a bit when I find the time and perhaps I'll be able to solve the mystery. Either way thanks for your time.

**Comment by xJon** [comment]
Closing due to inactivity (will reopen if requested!)

**Comment by xJon** [comment]
@Or-Gaule Can you try to manually add [this mod](https://modrinth.com/mod/tileentity-reset-fix/version/1.0.1-noreq) and see whether the issue is resolved?

**Comment by Or-Gaule** [comment]
Hello, I might be a bit late but I just wanted to let you know that I reinstalled the pack and somehow it's fixed now. Thanks
